boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -57,6 +57,11 @@
 	};
 };
 
+&rtc {
+	status = "okay";
+	#lora-rtc-cells = <0>;
```

Comment:
What is #lora-rtc-cells for?

Reply:
Because we had to use a phandle-array.

Reply:
I think what you have is phandles not a phandle-array.  phandle-array is a mixed type.